src/shared/sam/debugger/awsSamDebugger.ts

```diff
@@ -460,7 +460,7 @@ export class SamDebugConfigProvider implements vscode.DebugConfigurationProvider
         const handlerName = await getHandlerName(folder, config)
 
         config.baseBuildDir = resolve(folder.uri.fsPath, config.sam?.buildDir ?? (await makeTemporaryToolkitFolder()))
-        fs.ensureDir(config.baseBuildDir)
+        await fs.ensureDir(config.baseBuildDir)
```

Comment:
This fixes:

    rejected promise not handled within 1 second: Error: EACCES: permission denied, mkdir '/my'